devel/icu/Portfile

```diff
@@ -66,10 +66,19 @@ if {${subport} eq ${name} || ${subport} eq "${name}-lx"} {
     extract.suffix      .tgz
     worksrcdir          icu/source
 
+    # tar.gz from 74.2 is broken and contains symlink into ../LICENSE instead of LICENSE
+    # create a file which cintains link to license
```

Comment:
Spelling error. We don't necessarily have to fix it before we commit though.

Reply:
@pmetzger it was a copy and paste from `icu-devel` port. I suggest to keep it as is because fixing it on `icu-devel` means disabling CI because that port can't pass it :(

Reply:
Seems a bit silly but okay?